content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Cut Service Mesh Overhead with Istio Ambient Mesh"
+description: An anaylsis of how cloud resources are utilized in ambient and sidecar service mesh architectures.
+publishdate: 2023-07-14
+attribution: "Greg Hanson - Solo.io"
+keywords: [istio,ambient,waypoint,ztunnel]
+---
+
+Istio Ambient Mesh is the new Istio data plane without requiring sidecars, refer to the [announcement](/blog/2022/introducing-ambient-mesh/) for more details. Istio Ambient Mesh is included in [Istio 1.18](https://github.com/istio/istio/releases/tag/1.18.0) but is still considered to be in [alpha feature status](/docs/releases/feature-stages/#feature-phase-definitions).
+
+Sidecars have been a staple of Istio’s architecture since day one and are responsible for the majority of features available in Istio today. However, sidecars require the injection of an additional container to each Kubernetes pod resource, each of which needs allocated resources from the pod.
+
+Many of you are familiar with the simplified operation brought by the Istio ambient architecture. Let’s explore how ambient mesh cuts the service mesh costs typically associated with sidecars.
+
+## Check Out the Savings
+
+Our test scenario deploys four instances of the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo), each to their own namespace, with each underlying microservice scaled to 3 replicas. There will be a total of 48 application services and 144 applications pods in this setup. Test scenarios utilize the provided [loadgenerator](https://github.com/GoogleCloudPlatform/microservices-demo/tree/main/src/loadgenerator) microservice running in each of the namespaces to generate traffic during the observed window. The tests deployed Istio in several scenarios and resource usage and allocation are compared across the runs. The scenarios are:
+
+- Istio with sidecar
+- Ambient with L4 ztunnel only
+- Ambient with L4 ztunnel and L7 waypoint proxies (one per namespace, scaled to 3 replicas)
+
+Spoilers! There are significant savings across the board for all ambient scenarios when compared to sidecars – up to 99% savings with only L4 and 89% with L4+L7.
+
+{{< image width="100%"
+    link="savings.png"
+    caption="Total Memory and CPU: Consumption and Allocation"
+    >}}
+
+Looking at [total CPU and memory consumption numbers](#a-closer-look), we have to remember that in the [ambient architecture](/blog/2022/introducing-ambient-mesh/), there are no sidecar containers for every application pod in the mesh. The result is that memory usage of Istio’s dataplane in the ztunnel-only ambient scenario sees *more than 99% savings* compared to sidecar scenarios, and still 89% savings when waypoints are added. Looking at CPU, ztunnel once again sees 82% savings compared to the sidecar scenario, and 40% savings when waypoints are deployed.
```

Comment:
We led with "there are no sidecar containers", so this could be rewritten

```suggestion
Looking at [total CPU and memory consumption numbers](#a-closer-look), we see that we can remove the allocations required for a sidecar container in every application pod in the mesh. The result is that memory usage of Istio’s dataplane in the ztunnel-only ambient scenario sees *more than 99% savings* compared to sidecar scenarios, and still 89% savings when waypoints are added. Looking at CPU, ztunnel once again sees 82% savings compared to the sidecar scenario, and 40% savings when waypoints are deployed.
```